src/app/components/VideoPlayer/players.js

```diff
@@ -31,12 +41,13 @@ function _checkIfPlayersNeedToBeToggled(client) {
         ? proximityCheck(rect, client, player.willPlayAudio ? 0 : AMBIENT_PLAYABLE_RANGE)
         : proximityCheck(rect, client, (player.scrollplayPct || 0) / -100);
 
-    if (
-      (typeof player.isInPlayableRange === 'undefined' && isInPlayableRange) ||
-      (typeof player.isInPlayableRange !== 'undefined' && isInPlayableRange !== player.isInPlayableRange)
-    ) {
+    if (isInPlayableRange || isInPlayableRange !== player.isInPlayableRange) {
       enqueue(function _toggleVideoPlay() {
-        if (isInPlayableRange) {
+        let prefersReducedMotion =
+          document.body.classList.contains('is-reduced-motion') ||
+          (!document.body.classList.contains('is-high-motion') &&
+            window.matchMedia('(prefers-reduced-motion: reduce)').matches);
```

Comment:
Two things,
1. should this be a let? I can't see it being reassigned anywhere.
2. I think we can make the logic a bit clearer if we separate the condition out like so. What do you think?:

```suggestion
          const userPrefersReducedMotion =
            document.body.classList.contains("is-reduced-motion") ||
            document.body.classList.contains("is-high-motion")
              ? false
              : null;
          const prefersReducedMotion =
            userPrefersReducedMotion ?? window.matchMedia("(prefers-reduced-motion: reduce)").matches;
```

Interestingly, when you run each through the terser repl they compress down to the same line length so it's not impacting file size.